### PR TITLE
Add space between words 'our' and 'reference' in About

### PR DESCRIPTION
**Pull request by vegarsti**
Thanks for a very useful website. Just fixing this:

Screenshot 2023-02-07 at 14 32 23

### CODE REVIEW

File: pages/about.tsx [hunk 1]
```diff
@@ -258,6 +258,7 @@ const AboutPage = () => {
               to be paid upon execution, measured in gas. This cost is the same
               for all executions, though this is subject to change in new
               hardforks. See our
+              {' '}
```
Comment (peroket) [inline pages/about.tsx:261]: I think it has to go on the same line.
```suggestion
              hardforks. See our{' '}
```

**Review by peroket — CHANGES_REQUESTED** [review]
Thank you for finding and fixing that :-)
Just need to fix the syntax to compile. If you notice also there is another one just below with the same issue, would appreciate if you can add a space there too while you're at it.

**Comment by vegarsti** [comment]
Thanks! Like this?

**Review by peroket — APPROVED** [review]
Yes thanks a lot :-)

**Pull request merged**